nnpackage/schema/circle_schema.fbs

```diff
@@ -1521,7 +1521,13 @@ table InstanceNormOptions {
   fused_activation_function:ActivationFunctionType;
 }
 
+enum RoPEMode : int {
+  NEOX,
```

Comment:
@seanshpark 
when expressing in enum, should I exclude '0'?
the mode of test.recipe is set to 0. the option can be expressed abnormaly.
```
flatc --json ./res/CircleSchema/0.9/circle_schema.fbs -- ./build/compiler/common-artifacts/RoPE_000.circle

          builtin_options_type: "RoPEOptions",
          builtin_options: {
          }
```
if from '1', it is normal.   
and, Is there a missing code? Or does it not support 0 structurally?   
I thought you said that yesterday. I don't remember.--;

Reply:
>  should I exclude '0'?

I think we can follow what other enum does.
Others have explicitly have values like
```
  NEOX = 0,
```
And some has `NONE = 0` or `DEFAULT = 0`. But as I understand, we don't need none or default.

> and, Is there a missing code? Or does it not support 0 structurally?

I can't catch the problem right now. I'll look up to the code...

Reply:
@seanshpark 
thanks
i checked with circledump, and it is displayed normally
```
./build/compiler/circledump/circledump ./build/compiler/common-artifacts/RoPE_000.circle
O(0:0) ROPE
    mode(GPT_NEOX)
    I T(0:0) ifm
    I T(0:1) sin_table
    I T(0:2) cos_table
    O T(0:3) ofm
```